staging/src/k8s.io/apiextensions-apiserver/pkg/utils/utils.go

```diff
@@ -0,0 +1,84 @@
+package utils
```

Comment:
There is a helpers.go already. It would probably be better to move those functions here into the utils package.

Reply:
See my comment above.

Reply:
helpers have been moved back.